packages/webservice/app/src/main/resources/application.yaml

```diff
@@ -0,0 +1,39 @@
+elide:
+  modelPackage: 'com.yahoo.navi.ws.models.beans'
+  json-api:
+    path: /api/v1
+    enabled: true
+  graphql:
+    path: /graphql/api/v1
+    enabled: false
+  swagger:
+    path: /doc
+    enabled: true
+  async:
+    enabled: false
+
+spring:
+  application:
+    name: NaviWS
+  jpa:
+    hibernate:
+      show_sql: true
+      naming:
+        physical-strategy: 'org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl'
+      dialect: 'org.hibernate.dialect.H2Dialect'
```

Comment:
So you can add "profiles" to the application.yaml for different environments (like prod, test, etc). Is there any value in having something like that for the core repo?

Reply:
Yeah, I was planning to add template profiles. I don't want to make this PR too big. Made an issue for this: https://github.com/yahoo/navi/issues/864